prelude/__internal__/TestTools.mad

```diff
@@ -0,0 +1,53 @@
+import W from "Wish"
+import IO from "IO"
+import S from "String"
+import L from "List"
+import F from "Function"
+
+
+data AssertionError = AssertionError a a
+
+red :: String -> String
+red = (v) => ("\x1b[31m"++v++"\x1b[0m")
+
+green :: String -> String
+green = (v) => ("\x1b[32m"++v++"\x1b[0m")
```

Comment:
Totes not a blocker, just silly bikeshedding but since it's fun to think about things in terms of Madlib code:

```
colorString :: String -> String -> String
colorString = (color, v) => (color ++ v ++"\x1b[0m")

red = colorString("\x1b[31m")
green = colorString("\x1b[32m")
```

Reply:
Will do !